src/App.css

```diff
@@ -8,7 +8,8 @@
 
 .pac-container {
   z-index: 1501;
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New", monospace;
```

Comment:
Do we want to have the prettier changes done on a separate PR, rather than merging them in to this one?

Reply:
I think it's better to have them squashed into 1 PR so we can prevent drifting in case other PRs get submitted in between the prettier change PR, and GH action PR. The changes are in separate commits anyway, which aren't squashed so when we review commit history, it doesn't really matter.

 Happy to go either way. LMKWYT

Reply:
Fair argument, what do you think @noway